### PR TITLE
WIP - Metrics & Monitoring

### PR DESCRIPTION
**Pull request by becky-intelletive**

### CODE REVIEW

File: dash/backend/src/modules/metrics/services/m9sweeper_cron_jobs.service.ts [hunk 1]
```diff
@@ -1,41 +1,52 @@
...
         private readonly imageService: ImageService,
         private readonly exceptionsService: ExceptionsService,
     ) {}
 
     /**
-     * Recompile M9sweeper Metrics every 5 minutes for Prometheus metrics
+     * Recompile M9sweeper Metrics every 30s for Prometheus metrics
      */
-    @Cron('0 */5 * * * *')
+    @Cron('* * 01 * * *')
```
Comment (jacobbeasley) [inline dash/backend/src/modules/metrics/services/m9sweeper_cron_jobs.service.ts:22]: I think we discussed getting rid of this in leau of the new metrics API you are using. 

So then we'd have 2 metrics APIs: 
 - one that provides instance-level metrics (cpu usage, ram usage, response times, request volume, error rate)
 - One that provides env level details

So then this one would be supplanted by the new one that provides env level details.

Comment (becky-intelletive) [reply dash/backend/src/modules/metrics/services/m9sweeper_cron_jobs.service.ts:22]: Yes, that's the plan in the next PR :)

Comment (becky-intelletive) [reply dash/backend/src/modules/metrics/services/m9sweeper_cron_jobs.service.ts:22]: (Sorry in this PR not the next one - this is just a work in progress)

**Comment by jacobbeasley** [comment]
Sounds like this PR is no longer hte right - has been broken into 2 separate PRs

**Pull request closed without merging**